### ISSUE TITLE
Optimize inference time vs accuracy

### ISSUE THREAD

**Issue by knight-riders** [knight-riders/CarND-Capstone-Project#3]
Use Quantization, JIT compliation ... (see Lesson 11: Inference optimisation)

Experiment with heavier Models but compute the Region of interest dynamicaly in fonction of the car's position.

Traffic light coordinates are 3D, they include the z coordinate telling how high above the ground the center of the traffic light is.

http://xperroni.me/projecting-points-with-ros-pinhole-camera-model.html 
https://docs.opencv.org/2.4/modules/calib3d/doc/camera_calibration_and_3d_reconstruction.html
https://stackoverflow.com/questions/20573166/focal-length-to-field-of-view

There will be a static transform between base_link and camera published while running the code on the car. Here's the code for reference:
 arg name="period_in_ms" default="10"
arg name="cx" default="0.710706008953"
arg name="cy" default="-0.0233497393934"
arg name="cz" default="-0.318154357045"
arg name="cyaw" default="-1.5546953985199417"
arg name="cpitch" default="-0.005030400545831706"
arg name="croll" default="-1.4230227212132798"
arg name="cframe_id" default="velodyne"
arg name="cchild_frame_id" default="camera"
node pkg="tf" type="static_transform_publisher" name="localizer_to_camera" args="$(arg cx) $(arg cy) $(arg cz) $(arg cyaw) $(arg cpitch) $(arg croll) $(arg cframe_id) $(arg cchild_frame_id) $(arg period_in_ms)"

Requirements:
TensorFlow v1.3, Keras v2.0.8) + Cuda 8 + Cudnn 6.0
